app/Http/Middleware/Authenticate.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Middleware;
+
+use Illuminate\Auth\Middleware\Authenticate as Middleware;
+use Illuminate\Http\Request;
+
+class Authenticate extends Middleware
+{
+    /**
+     * Get the path the user should be redirected to when they are not authenticated.
+     */
```

Comment:
These unnecessary comments could be removed with codestyle comment purge if my PR would be finally accepted: https://github.com/blumilksoftware/codestyle/pull/86

Reply:
i left comment for u